src/views/CatsView.vue

```diff
@@ -4,17 +4,15 @@
       <h1>
         {{ i18n.t("cats") }}
       </h1>
-      <v-input
+      <v-search
         class="cats-view__search"
         :label="i18n.t('search')"
         v-model="search"
-      ></v-input>
+      ></v-search>
 
-      <v-button class="cats-view__add-btn button--icon">
-        <template #icon>
-          <wc-svg-icon href="/icons.svg" symbol="add"></wc-svg-icon>
-        </template>
-      </v-button>
+      <wc-button size="big" class="cats-view__add-btn" iconOnly="true">
```

Comment:
Looks like we can pass boolean props without specifying their 'true' values
```suggestion
      <wc-button size="big" class="cats-view__add-btn" iconOnly>
```